docs/dev/python/user-guide/datastream/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "DataStream API"
+nav-id: python_datastream_api
+nav-parent_id: python_user_guide
+nav-pos: 30
+nav-show_overview: true
```

Comment:
```suggestion
```

Reply:
I don't think we need this page. Let's revisit this after the rest of the content is merged in.

Reply:
ok, we can add this after all content are ready.